compiler/rustc_interface/src/passes.rs

```diff
@@ -795,6 +793,8 @@ pub fn create_global_ctxt<'tcx>(
                 untracked,
                 dep_graph,
                 queries.on_disk_cache.as_ref().map(OnDiskCache::as_dyn),
```

Comment:
side note: is there a reason we don't make this a method on `QueryEngine`? seems weird to pass in a reference to the query engine and to a field of it.